server/historian/packages/historian-base/src/routes/utils.ts

```diff
@@ -104,6 +108,16 @@ export async function createGitService(createArgs: createGitServiceArgs): Promis
 	const customData: ITenantCustomDataExternal = details.customData;
 	const writeToExternalStorage = !!customData?.externalStorageData;
 	const storageUrl = config.get("storageUrl") as string | undefined;
+
+	let isEphemeral = isEphemeralContainer;
+	if (!ignoreEphemeralFlag) {
```

Comment:
In general, I think this whole `ignoreEphemeralFlag` is adding a lot of branching logic that could be avoided by having the ephemeralDoc feature flag at the Gitrest level. Let Historian be a pass-through service that handles what to tell Gitrest. Let Gitrest determine how to handle that information.

Reply:
we need this at the historian level so that we can skip the caching..that is why we have this config in historian instead of gitrest